### PR TITLE
Make Value a number

### PR DESCRIPTION
**Pull request by quisar**
This allows to use Value as a number in typescript without generating warnings.

### CODE REVIEW

File: behavior.d.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 
-declare class Value {
-  num: number;
+type Value = number & {
```
Comment (ribrdb) [inline behavior.d.ts:2]: Oh, nice idea!

File: behavior.d.ts [hunk 2]
```diff
@@ -207,7 +206,7 @@ declare function getCompReg(component_index: Value | CompNum, group_index?: Valu
 /**
  * Sets the numerical/coordinate part of a value
  */
-declare function setNumber(value: Value, num_coord: Value | CoordNum): Value;
```
Comment (ribrdb) [inline behavior.d.ts:210]: Why do you need this change?

Comment (quisar) [reply behavior.d.ts:210]: Because I want to be able to call
foo = setNumber(bar, 3);

Comment (ribrdb) [reply behavior.d.ts:210]: Isn't that already possible? 
```ts
type CoordNum = Coord | number;
```

Comment (quisar) [reply behavior.d.ts]: You are right, I missed something. Removed.

**Review by ribrdb — APPROVED** [review]

**Pull request merged**